src/langs/fr.js

```diff
@@ -48,5 +48,9 @@ export default {
     'sorted ascending': 'trié par ordre croissant',
     "There isn't any question to display":
       "Il n'y a aucune questions à afficher",
+    'Results by question': 'Résultats par question',
+    'Results by user': 'Résultats par utilisateur',
```

Comment:
Just thinking, should it be `results` or `answers`? What do you think fit better?

Reply:
I think I would rather keep `results` as we display more info than just the answer. Do you have a preference ?